runtime/development/src/lib.rs

```diff
@@ -1927,14 +1927,14 @@ construct_runtime!(
 		Investments: pallet_investments::{Pallet, Call, Storage, Event<T>} = 105,
 		LiquidityRewardsBase: pallet_rewards::<Instance1>::{Pallet, Storage, Event<T>, Config<T>} = 106,
 		LiquidityRewards: pallet_liquidity_rewards::{Pallet, Call, Storage, Event<T>} = 107,
-		Connectors: pallet_connectors::{Pallet, Call, Storage, Event<T>} = 108,
+		LiquidityPools: pallet_liquidity_pools::{Pallet, Call, Storage, Event<T>} = 108,
```

Comment:
Without a migration, this does not automatically translate `Connectors` prefixed storage entries to `LiquidityPools`. So we either need to write one or live with rebuilding in the LiquidityPools testnet.

Reply:
I am aware but we agreed to not write migrations for `development`, right? and when we deploy the next test revision we will have the new setup with the new gateway and routers setup so we would need to set storage again anyways.

Reply:
Perfect! Just wanted to raise awareness as this might have been a rare exception in which we might have had to write a migration. Glad to hear we will restart fgrom scratch anyway.

Reply:
Definitely, this should **never** happen in production of course. Thanks for raising it anyways!